src/components/forms/TransitGatewayForm.js

```diff
@@ -51,29 +55,50 @@ class TransitGatewayForm extends Component {
     this.setState(this.eventTargetToNameAndValue(event));
   }
 
+  /**
+   * Handle crn input
+   * @param {event} event
+   */
+  handleCRNs(event) {
+    invalidRegex;
+    let crnList = event.target.value
+      ? event.target.value
+          .replace(/\s\s+/g, "") // replace extra spaces
+          .replace(/,(?=,)/g, "") // prevent null tags from
+          .replace(/[^\w,-:]/g, "")
+          .split(",")
+      : [];
+    this.setState({ crns: crnList });
+  }
+
+  /**
+   * Invalid crns
+   * @param {Array} crns
+   */
+  // https://cloud.ibm.com/docs/account?topic=account-crn
+  invalidCRNs(crns) {
+    if (crns.length === 0) return false;
+    for (let crn of crns) {
+      if (crn.match(this.props.crnRegex) === null) {
+        return true;
+      }
+    }
+    return false;
+  }
+
   render() {
     return (
       <>
         <IcseFormGroup>
-          <IcseToggle
-            labelText="Global"
-            toggleFieldName="global"
-            id={this.state.name + "-tg-global"}
-            onToggle={this.handleToggle}
-            defaultToggled={this.state.global}
-          />
           <IcseTextInput
             onChange={this.handleInputChange}
             componentName="Transit Gateway"
             field="name"
             value={this.state.name}
-            readOnly={this.props.readOnlyName}
+            readOnly={true}
```

Comment:
name should not be readOnly - we need ability to create multiple tgw
there was a usecase for the prop too but i dont remember it off the top of my head

Reply:
I'll put the prop back then! ◡̈

Reply:
Checking in CRAIG, we aren't ever setting readOnlyName to true, so it's never being used. So I think we could probably remove the prop completely, I don't think it should ever be readOnly

but i am like 99% sure we had some reason for having the prop that I just can't remember.. @jvallexm ?

Reply:
taking away readOnly ◡̈